test/integration.sh

```diff
@@ -77,10 +91,54 @@ if ./tmp/bin/up \
   --labels='_id="test"' \
   --token="$token"; then
   result=0
-  printf "\t## tests: ok"
-  exit 0
+  printf "\t## metrics tests: ok\n\n"
+else
+  result=1
+  printf "\t## metrics tests: failed\n\n"
+  exit 1
 fi
 
-printf "\t## tests: failed" 1>&2
-result=1
-exit 1
+echo "-------------------------------------------"
+echo "- Logs Write tests                        -"
+echo "-------------------------------------------"
+
+if curl \
+     -v \
```

Comment:
do we need to add `--fail` to these curl requests? otherwise if there is a 404 or 500 or something, this will still exit 0. Also, do we need to somehow test the content of the response? how do we know that we got back what we were expecting?

Reply:
`--fail` is a good hint. However, checking response content is way to much a simple integration test that assert proxying requests/responses. I am not aware if `observatorium/up` has the capability to assert such things. If yes, I would be glad to contribute it there instead of solving this in bash. WDYT?

Reply:
Agreed. Up doesn't check the content against an expected response but it DOES ensure the status code is 200